src/fides/api/api/v1/endpoints/privacy_request_endpoints.py

```diff
@@ -2592,6 +2593,11 @@ def get_access_results_urls(
     """
     Endpoint for retrieving access results URLs for a privacy request.
     """
+    if os.environ.get("SECURITY__SUBJECT_REQUEST_DOWNLOAD_UI_ENABLED") != "true":
```

Comment:
This all works as expected but you should use our existing pattern for configs instead of creating a new env variable. I agree that this should be a security setting so look into adding it here https://github.com/ethyca/fides/blob/LA-89-add-connection-retries-to-database-tasks/src/fides/config/security_settings.py#L24. The end result is we should be able to do this
```suggestion
    if not config.security.subject_request_download_ui_enabled:
```